file_optical.c

```diff
@@ -233,6 +233,7 @@ static int fbo_emulate_request_sense(struct tcmu_device *dev, uint8_t *cdb,
 {
```

Comment:
For this patch we will do the linux kernel'ish style of get/put be* in this PR:

https://github.com/open-iscsi/tcmu-runner/pull/282